static/js/click_handlers.js

```diff
@@ -556,18 +557,28 @@ export function initialize() {
         });
 
     function do_render_buddy_list_tooltip(elem, title_data) {
-        elem.tooltip({
-            template: render_buddy_list_tooltip(),
-            title: render_buddy_list_tooltip_content(title_data),
-            html: true,
-            trigger: "hover",
-            placement: "bottom",
-            animation: false,
-        });
-        elem.tooltip("show");
+        let placement = "left";
+        if (window.innerWidth < Number(media_breakpoints.md_min.slice(0, -2))) {
+            // On small devices display tooltips based on available space.
+            // It default to `bottom` on its own.
+            placement = "auto";
+        }
+        tippy(elem[0], {
+            // Quickly display and hide tooltips
+            // so that they don't stick and overlap
+            // each other.
+            delay: 0,
 
-        $(".tooltip").css("left", elem.pageX + "px");
-        $(".tooltip").css("top", elem.pageY + "px");
+            content: render_buddy_list_tooltip_content(title_data),
+            arrow: true,
+
+            // keeping placement `left` doesn't hide other users in
+            // the list.
+            placement,
+
+            allowHTML: true,
```

Comment:
We can probably remove most of this whitespace, right?

Reply:
We can, but having a new line here make clear that the comment above is for `placement` only.